problems/1.types+operators.js

```diff
@@ -2,15 +2,17 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  (val % 2 === 0) || (val % 2 === 0 )
+  return true 
 }
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
+function isTruthy(val) {
+  
```

Comment:
YOu're being given an argument. You have to check for truthyness. Remember the only falsey values in JS are 0, "",  null, undefined, NaN, false.